lib/dynflow/coordinator.rb

```diff
@@ -0,0 +1,273 @@
+require 'dynflow/coordinator_adapters'
+
+module Dynflow
+  class Coordinator
```

Comment:
I would like to add more documentation to the pages, and then I will add short description here, with linking to the more comprehensive documentation in the doc directory